tests/config_id/enc/props_kss.c

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+
+OE_SET_ENCLAVE_SGX2(
+    1,    /* ProductID */
+    1,    /* SecurityVersion */
+    {0},  /* ExtendedProductID */
+    {0},  /* FamilyID */
+    true, /* Debug */
+    true, /* CapturePFGPExceptions */
```

Comment:
minor: setting `CapturePFGPExceptions` to false so that this tests focus only on the `RequireKSS`